medusa/storage/google_storage.py

```diff
@@ -30,6 +30,9 @@
 from medusa.storage.google_cloud_storage.gsutil import GSUtil
 
 
+GSUTIL_MAX_FILES_PER_CHUNK = 64
```

Comment:
This doesn't belong in this PR. Either we have a missing commit in `master` or this PR should be properly rebased against master.
I'll check what's wrong here (and the rebase is needed anyway).

Reply:
@jsanda - Alex understood where this was coming from and fixed it. Rebase should do now that we updated master.